.github/workflows/solidity-exporter.yml

```diff
@@ -0,0 +1,50 @@
+name: Export Interfaces And Contracts
+
+on: [push]
+
+jobs:
+  export:
+    name: Generate Interfaces And Contracts
+    runs-on: ubuntu-latest
```

Comment:
Instead of adding the comment on the bottom, couldn't we add an `if: false` on the job itself? That way it will show as skipped in the CI until they enable it.